int32 id
float64 minX
float64 minY
float64 maxX
float64 maxY
string label